### PR TITLE
Resolved #14

### LINKED ISSUE #14: Add the Hack club rait logo
Hii folk's. 
**In this issue You  are required to make the lable's of  for contributor , repo size , open issue etc as shown below  and Also add the logo of the HACK CLUB RAIT . as shown below after the cover image !**
<br>
Screenshot from 2022-07-26 23-13-06

**The logo of the HACK CLUB RAIT IS ATTACHED BELOW** 

**PS : Adjust the height and width accordingly .**

### PR DESCRIPTION
**Pull request by khushimarothi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,19 @@ This repo will have web3 project which consist of the smart contract(backend) an
...
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+![GitHub watchers](https://img.shields.io/github/watchers/Vikash-8090-Yadav/Web3.0-World)
+![GitHub forks](https://img.shields.io/github/forks/Vikash-8090-Yadav/Web3.0-World)
+![GitHub Repo stars](https://img.shields.io/github/stars/Vikash-8090-Yadav/Web3.0-World)
 ![GitHub contributors](https://img.shields.io/github/contributors/Vikash-8090-Yadav/Web3.0-World)
-
+![GitHub last commit](https://img.shields.io/github/last-commit/Vikash-8090-Yadav/Web3.0-World)
+  
 ![GitHub repo size](https://img.shields.io/github/repo-size/Vikash-8090-Yadav/Web3.0-World)
+
```
Comment (Vikash-8090-Yadav) [inline README.md:22]: please include the License label

File: README.md [hunk 2]
```diff
@@ -108,9 +108,8 @@ Show some ❤️&nbsp; by giving <img src="https://imgur.com/o7ncZFp.jpg" height
  # Part of Open Source Program
```
Comment (Vikash-8090-Yadav) [inline README.md:108]: Please include  all that  label  and logo section  after the cover image  respectively . 
Thank u :smile:

File: README.md (hunk 1 above)
Comment (khushimarothi) [reply README.md:22]: The license label that I'm creating shows that the license for this repo is not specifed.   Will attach the ss soon. Also there is no issue in the pull request part.

File: README.md [hunk 3]
```diff
@@ -10,10 +10,19 @@ This repo will have web3 project which consist of the smart contract(backend) an
   <div align="center">
  <p>
```
[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md:12]: Okay will do that

File: README.md (hunk 1 above)
Comment (Vikash-8090-Yadav) [reply README.md:22]: Ya i applied the MIT license please check

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md:6]: I tried everything but i am not getting why its fetching value of total lines as invalid

**Comment by khushimarothi** [comment]
let me know if the suggested changes are visible...

**Comment by Vikash-8090-Yadav** [comment]
Fetch the latest changes as your branch is 3 commit behind from mine .

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vikash-8090-Yadav) [reply README.md:6]: include this in place of total lines
<a href="https://github.com/Vikash-8090-Yadav/Web3.0.World">LOC</a> &nbsp;

**Comment by khushimarothi** [comment]
yes done

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vikash-8090-Yadav) [reply README.md:6]: > I tried everything but i am not getting why its fetching value of total lines as invalid

OK  fine , delete Line of code label
 we ll see later

[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md:6]: okay

File: README.md [hunk 4]
```diff
@@ -25,8 +25,9 @@
  <h3 align="center"> Part of Open Source Program</h3>
  <div align="center">
  <a href="https://soc.hackclubrait.co/">Hack Club RAIT SOC</a>
```
Comment (Vikash-8090-Yadav) [inline README.md:28]: add HSOC link under the image only .

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md:6]: Screenshot (71)
Im able to see loc now

File: README.md (hunk 4 above)
Comment (khushimarothi) [reply README.md:28]: under the logo right???

File: README.md [hunk 5]
```diff
@@ -25,8 +25,9 @@
  <h3 align="center"> Part of Open Source Program</h3>
```
Comment (Vikash-8090-Yadav) [inline README.md:26]: Remove line 25 and add this line 
 ## 📌 Opensource Programs


### This project is a part of following Open Source Programs

PS : **THE space between the above two is required don't change that**

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vikash-8090-Yadav) [reply README.md:6]: > Screenshot (71) Im able to see loc now

but afrer clicking on LOC it is showing error

[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md:6]: okay i have removed it for now.

File: README.md (hunk 5 above)
Comment (khushimarothi) [reply README.md]: check if it is fine now

File: README.md [hunk 6]
```diff
@@ -22,12 +22,13 @@
  </p>
  </div>
  
- <h3 align="center"> 📌 Open Source Program</h3>
```
[the remark this answers is not in the document]
Comment (Vikash-8090-Yadav) [reply README.md]: @khushimarothi see this ..

[the remark this answers is not in the document]
Comment (Vikash-8090-Yadav) [reply README.md]: > on 25 line and afterwards **These two should align to the left not center . please note that and logo should be in centre only** Copy the code under the " " only and don't change the space
> 
> " ## pushpin Opensource Programs"
> 
> "### This project is a part of following Open Source Programs"
> 
> PS : **THE space between the above two is required don't change that **

@khushimarothi see this .. 
and remove the link fror the HSSOC also

[the remark this answers is not in the document]
Comment (khushimarothi) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -22,15 +22,15 @@
...
- <h2 align="center"> 📌 Open Source Program</h2>
- <div align="center">
- <h4 >This project is a part of following Open Source Programs</h2>
+ ## 📌 Open Source Program
+
+ ### This project is a part of following Open Source Programs
 
+ <div align="center">
 
  ![logo](https://user-images.githubusercontent.com/85816852/181296374-a2273b6c-45af-45d1-b5d8-ee871a035c1f.jpg)
```
Comment (Vikash-8090-Yadav) [inline README.md:31]: All look's great ..do these changes only :smile: 
**add this  logo to the image folder  created and increase the width=55% and adjust the height accordingly . If necessary change  the width also .**

Comment (khushimarothi) [reply README.md]: 55% was too big I adjusted it to 20% now.

Comment (Vikash-8090-Yadav) [reply README.md]: > 55% was too big I adjusted it to 20% now.

ok no problem

**Comment by Vikash-8090-Yadav** [comment]
Great

**Pull request merged**